virt/virtual_machines/vm_networking/virt-configuring-certificate-rotation.adoc

```diff
@@ -0,0 +1,9 @@
+[id="virt-configuring-certificate-rotation"]
+= Configuring certificate rotation
+include::modules/virt-document-attributes.adoc[]
+:context: virt-configuring-certificate-rotation
+
+toc::[]
+
```

Comment:
It's generally mod docs practice to include a short intro in the assembly (before the include files). A sentence is usually good enough :)